### PR TITLE
Minor tweaks to Geography API types & Geography Service

### PR DESCRIPTION
**Pull request by avatarneil**
## 📚 Purpose
Minor tweaks to the Geography API GET Geographies return type, and the GeographyService to allow bypassing RPC.

## 📦 Impacts:
- mds-geography-api
- mds-geography-service

### CODE REVIEW

File: packages/mds-geography-service/@types/index.ts [hunk 1]
```diff
@@ -54,8 +54,8 @@ export type GetPublishedGeographiesOptions = GetGeographiesOptions &
     publishedAfter: Timestamp
   }>
 
-export type GeographyWithMetadataDomainModel<M extends {} = {}> = GeographyDomainModel &
-  Partial<Pick<GeographyMetadataDomainModel<M>, 'geography_metadata'>>
+export type GeographyWithMetadataDomainModel<M extends Record<string, any> = Record<string, any>> =
```
Comment (avatarneil) [inline packages/mds-geography-service/@types/index.ts:57]: I don't love this type, would be open to something which makes things a bit more optional.

Comment (PeterSP) [reply packages/mds-geography-service/@types/index.ts:57]: What's the difference between Record<string, any> and {}?

**Review by PeterSP — APPROVED** [review]

File: packages/mds-geography-service/@types/index.ts (hunk 1 above)
Comment (avatarneil) [reply packages/mds-geography-service/@types/index.ts:57]: `{}` is a literal empty object, with no properties at all. Record<string, any> is a fair bit more lenient

File: .changeset/many-dingos-cry.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@mds-core/mds-geography-service": patch
+---
+
+Make getPublishedGeographies bypass RPC by default, adjust GeographyWithMetadataDomainModel to be more accurate
```
Comment (drtyh2o) [inline .changeset/many-dingos-cry.md:5]: Bypassing RPC should _not_ be the _default_ behavior; it should be the exception.

**Pull request merged**